### ISSUE TITLE
Major issue on Android 13

### ISSUE THREAD

**Issue by Pakodabolte** [weihuoya/citra#721]
So i first found this issue trying to play super mario 3d world after updating to Android 13 I don't know how to describe this issue it makes many games unplayable I don't know if it is a Android side of thing for emulator but [Video](https://drive.google.com/file/d/14DQd1_xVNQPz2J7xodY4PJ1OSoG2eTxe/view?usp=drivesdk) watch the video from the link to understand it and please fix this issue

**Comment by Sango916** [comment]
In #706 I thought this Citra glitch was caused by the Pixel 7 Pro, but maybe it is caused by Android 13... do you experience similar glitches on your devices?

**Comment by Pakodabolte** [comment]
> 

Yes I'm experiencing this issue on my vivo v25 after updating to Android 13 there was nothing like this when i was on 12 and used the same citra version

**Comment by Pakodabolte** [comment]
> In #706 I thought this Citra glitch was caused by the Pixel 7 Pro, but maybe it is caused by Android 13... do you experience similar glitches on your devices?

This also happens in the official citra i tried it today hoping i could play some games but nope

**Comment by Sango916** [comment]
The official version hasn't been updated for a long time... It's a natural result.

**Comment by Pakodabolte** [comment]
> The official version hasn't been updated for a long time... It's a natural result.

Uh no I'm talking about the universal official version not the Android builds 
Nightly and Canary version which you can get from their website it updated yesterday i think

**Comment by Pakodabolte** [comment]
> The official version hasn't been updated for a long time... It's a natural result.

This doesn't only happen in citra but in dolphin as well but it isn't as much as citra

**Comment by Sango916** [comment]
Dolphin is regularly updated on the Play Store, but Citra is not.

Sorry for the machine translation on my part.

**Comment by Pakodabolte** [comment]
> Dolphin is regularly updated on the Play Store, but Citra is not.
> 
> Sorry for the machine translation on my part.

I was not talking about the playstore version see the previous reply i forgot to quote my bad

**Comment by Sango916** [comment]
Both have updates distributed on their official sites, but I was able to find out that only Dolphin's update is reflected in the Play Store.
